libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -310,123 +143,61 @@ - (void)enumerateDelegates:(void (^)(id<SFUserAccountManagerDelegate>))block
     }
 }
 
-#pragma mark - Temporary User
-
-- (SFUserAccount *)temporaryUser {
-    SFUserAccount *tempAccount = (self.userAccountMap)[self.temporaryUserIdentity];
-    if (tempAccount == nil) {
-        tempAccount = [self createUserAccount];
-    }
-    return tempAccount;
-}
-
 #pragma mark - Anonymous User
-
-+ (BOOL)isUserAnonymous:(SFUserAccount*)user {
-    if (nil == user.accountIdentity) {
-        return NO;
-    }
-    return [user.accountIdentity.userId isEqualToString:SFUserAccountManagerAnonymousUserAccountUserId] &&
-        [user.accountIdentity.orgId isEqualToString:SFUserAccountManagerAnonymousUserAccountOrgId];
-}
-
-- (BOOL)supportsAnonymousUser {
-    return [[[NSBundle mainBundle] objectForInfoDictionaryKey:kSFUserAccountSupportAnonymousUsage] boolValue];
-}
-
-- (BOOL)autocreateAnonymousUser {
-    return [[[NSBundle mainBundle] objectForInfoDictionaryKey:kSFUserAccountAutocreateAnonymousUser] boolValue];
-}
-
 - (BOOL)isCurrentUserAnonymous {
-    return [[self class] isUserAnonymous:self.currentUser];
+    return self.currentUser == nil;
 }
 
-- (SFUserAccount *)anonymousUser {
-    if (nil == _anonymousUser) {
-        for (SFUserAccountIdentity *identity in self.allUserIdentities) {
-            if ([identity isEqual:self.anonymousUserIdentity]) {
-                _anonymousUser = [self userAccountForUserIdentity:identity];
-                break;
-            }
-        }
+-(NSMutableDictionary *)userAccountMap {
+    if(!_userAccountMap || _userAccountMap.count < 1) {
+        [self reload];
     }
-    return _anonymousUser;
+    return _userAccountMap;
 }
 
-- (void)setupAnonymousUser:(BOOL)supportsAnonymousUser autocreateAnonymousUser:(BOOL)autocreateAnonymousUser {
-
-    // If there is no current user but the application support anonymous user
-    // and wants it to be created automatically, then create it now.
-    if (supportsAnonymousUser && autocreateAnonymousUser && nil == self.anonymousUser) {
-        [self log:SFLogLevelInfo msg:@"Creating anonymous user"];
-        [self enableAnonymousAccount];
-        if (nil == self.currentUser) {
-            self.currentUser = self.anonymousUser;
-        }
-    }
+- (void)setUserAccountMap:(NSMutableDictionary *) userAccountMap {
+    _userAccountMap = userAccountMap;
```

Comment:
Actually, scratch that: you don't need the custom setter at all, as iOS will automatically provide the given implementation.  This can be removed.